platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
If it shows `validation-warnings` and `validation-errors` like it advertises, then I agree, Metacard is a better name. The warnings and errors are a part of metacards, not just metadata.